INSTALL.md

```diff
@@ -30,9 +30,10 @@ If you'd like to create a VM with BLT, you will require the following additional
 
         brew tap caskroom/cask
         brew install ansible
+        brew upgrade ansible
         brew cask install virtualbox vagrant
 
-The minimum required versions are VirtualBox 5.1.x and Vagrant 1.8.6. 
+The minimum required versions are VirtualBox 5.1.x, Vagrant 1.8.6 and Ansible 2.2.x. 
```

Comment:
Ansible isn't technically required at all—if not installed locally, it will be installed at the right version inside the VM (and this is preferable on non-unixy environments). I'd rather not add it as a requirement in the BLT docs.